### PR TITLE
Remove ExternalToolManager

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-14562
affects: student, teacher
release note: none

Test plan:
- Launch an LTI tool from Course Nav
  - narmstrong > s > CS 1400 > Google Drive (it should ask for auth)
- Launch an LTI tool from a link in a page
  - narmstrong > s > CS 1400 > Pages > Links to Stuff > External Tool

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/ck3yquf) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fck3yquf)
teacher: [Link](https://rebrand.ly/0nfom01) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F0nfom01)
parent: [Link](https://rebrand.ly/k1v5akd) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fk1v5akd)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a reference to a jira ticket. For example: `refs: MBL-10023`
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.42% | 93.43% | -0%

[MBL-14562](https://instructure.atlassian.net/browse/MBL-14562)

#### Affected Apps: Student, Teacher
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Routes.swift [hunk 1]
```diff
@@ -184,9 +184,10 @@ let routeMap: KeyValuePairs<String, RouteHandler.ViewFactory?> = [
     "/:context/:contextID/discussion_topics/:discussionID": discussionViewController,
 
     "/courses/:courseID/external_tools/:toolID": { url, params, _ in
-        guard let url = url.url, let session = Session.current else { return nil }
+        guard let id = params["toolID"] else { return nil }
         guard let vc = HelmManager.shared.topMostViewController() else { return nil }
-        ExternalToolManager.shared.launch(url, in: session, from: vc, fallbackURL: url)
+        let tools = LTITools(id: id)
```
Comment (vanwagonet) [inline Student/Student/Routes.swift:189]: should this have the context set to the course as well?

Comment (armstrongnate) [reply Student/Student/Routes.swift:189]: definitely won't hurt. I'll add it.

**Pull request merged**